examples/browser/package.json

```diff
@@ -48,6 +48,7 @@
     "@theia/search-in-workspace": "^1.2.0",
     "@theia/task": "^1.2.0",
     "@theia/terminal": "^1.2.0",
+    "@theia/timeline": "^1.2.0",
```

Comment:
Let's keep the browser app in sync with the electron one.

Reply:
done